### PR TITLE
Seeded database and added Faker

### PR DESCRIPTION
**Pull request by advantagevarnson**

### CODE REVIEW

**Comment by advantagevarnson** [comment]
@droberts84

File: db/seeds.rb [hunk 1]
```diff
@@ -7,14 +7,29 @@
...
+  5.times do
+    user = User.create!(
+      name: Faker::Name.name,
+      email: Faker::Internet.email,
+      password: Faker::Lorem.characters(10)
+    )
+    user.skip_confirmation!
+    user.save!
+  end
+  users = User.all
```
Comment (droberts84) [inline db/seeds.rb]: On lines 14-18, you are saving each `user` to the database with `User.create`, then on this line you query the database to find all of the users you just created above.

Instead you could make a slight tweak to keep a reference to every user you are saving to the database, so you don't have to query again to get that information.

Something like

```ruby
users = 5.times.map do
  user = User.create!(....)
  ...
  user.save!
  user
end
```

Recall that the `map` method returns an array containing whatever was returned from each iteration of the block.

File: db/seeds.rb [hunk 2]
```diff
@@ -7,14 +7,29 @@
...
+
+  #Create items
+  25.times do
+    Item.create!(
+      user: users.sample,
+      name: Faker::Lorem.sentence
+    )
+  end
+  items = Item.all
```
Comment (droberts84) [inline db/seeds.rb]: you could do the same here

**Comment by droberts84** [comment]
:+1:

**Pull request closed without merging**